src/main/java/nl/hsac/fitnesse/junit/HsacFitNesseRunner.java

```diff
@@ -136,6 +140,33 @@ protected void runPages(List<WikiPage> pages, RunNotifier notifier) {
 
     }
 
+    // In the original runner class, we cannot use a system property as annotation value,
+    // so we override it to work exactly like the suiteFilter annotation.
+    // This also needs the ExcludeSuiteFilter interface
+    @Override
+    protected String getExcludeSuiteFilter(Class<?> klass) throws Exception {
+        ExcludeSuiteFilter excludeSuiteFilterAnnotation = (ExcludeSuiteFilter)klass.getAnnotation(ExcludeSuiteFilter.class);
+        if (excludeSuiteFilterAnnotation == null) {
```

Comment:
I've created a pull request to FitNesse project for the exclude filter based on annotations https://github.com/unclebob/fitnesse/pull/1033